Shaders/deferred_light/deferred_light.frag.glsl

```diff
@@ -513,6 +535,17 @@ void main() {
 		);
 	}
 #endif // _Clusters
-
 	fragColor.a = 1.0; // Mark as opaque
+
+	// Show voxels
+	/*
+	vec3 origin = vec3(texCoord * 2.0 - 1.0, 0.99);
+	vec3 direction = vec3(0.0, 0.0, -1.0);
+	vec4 color = vec4(0.0f);
+	for(uint step = 0; step < 400 && color.a < 0.99f; ++step) {
+	 	vec3 point = origin + 0.005 * step * direction;
+	 	color += (1.0f - color.a) * textureLod(voxels, point * 0.5 + 0.5, 0);
+	}
+	fragColor.rgb += color.rgb;
+	*/
```

Comment:
Is this for debugging (👍 )? If so, what do you think of adding a `#define DEBUG_VOXELS 1` at the top of the shader so that it can be easily toggled?

Reply:
Yes that is for debug, no It's better leave it like this. Lubos had used that before when creating the shaders I guess.